src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  switch (true) {
+    case (days >= 7):
```

Comment:
Let's create the variable for each condition (days >= 7, days >=3)

Reply:
Thx, got it